session/session.go

```diff
@@ -2011,23 +2011,27 @@ func BootstrapSession(store kv.Storage) (*domain.Domain, error) {
 		}
 	}
 
-	err = executor.LoadExprPushdownBlacklist(se)
+	se4, err := createSession(store)
```

Comment:
Could you add some comments?

Reply:
A few lines above the code in this function already have the comment

https://github.com/pingcap/tidb/pull/21421/files/4e36b506353d62f8c0a31c727d28c352b15efb8d#diff-10dab83c04a99358d431906d37f48b92be97d50a2719d4ddf81c62d3b03a0aebR1992-R1994